zeebe/exporters/rdbms-exporter/src/main/java/io/camunda/exporter/rdbms/ProcessInstanceExportHandler.java

```diff
@@ -38,33 +38,15 @@ public void export(final Record<ProcessInstanceRecordValue> record) {
     if (record.getIntent().equals(ProcessInstanceIntent.ELEMENT_ACTIVATING)) {
       processInstanceWriter.create(map(record));
     } else if (record.getIntent().equals(ProcessInstanceIntent.ELEMENT_COMPLETED)) {
-      processInstanceWriter.update(
-          new ProcessInstanceDbModel(
-              value.getProcessInstanceKey(),
-              value.getBpmnProcessId(),
-              value.getProcessDefinitionKey(),
-              ProcessInstanceState.COMPLETED,
-              null,
-              DateUtil.toOffsetDateTime(Instant.ofEpochMilli(record.getTimestamp())),
-              value.getTenantId(),
-              value.getParentProcessInstanceKey(),
-              value.getParentElementInstanceKey(),
-              null,
-              value.getVersion()));
+      processInstanceWriter.end(
+          value.getProcessInstanceKey(),
+          ProcessInstanceState.COMPLETED,
+          DateUtil.toOffsetDateTime(Instant.ofEpochMilli(record.getTimestamp())));
     } else if (record.getIntent().equals(ProcessInstanceIntent.ELEMENT_TERMINATED)) {
-      processInstanceWriter.update(
-          new ProcessInstanceDbModel(
-              value.getProcessInstanceKey(),
-              value.getBpmnProcessId(),
-              value.getProcessDefinitionKey(),
-              ProcessInstanceState.CANCELED,
-              null,
-              DateUtil.toOffsetDateTime(Instant.ofEpochMilli(record.getTimestamp())),
-              value.getTenantId(),
-              value.getParentProcessInstanceKey(),
-              value.getParentElementInstanceKey(),
-              null,
-              value.getVersion()));
+      processInstanceWriter.end(
+          value.getProcessInstanceKey(),
+          ProcessInstanceState.CANCELED,
+          DateUtil.toOffsetDateTime(Instant.ofEpochMilli(record.getTimestamp())));
```

Comment:
I created in the DateUtil class (on the variables PR) a function which also does the ofEpochMilli stuff, since we always use it like this.

Reply:
❓Where? I can't find it in the PR

Reply:
A is in the userTask branch. 

  public static OffsetDateTime toOffsetDateTime(final Long timestamp) {
    return OffsetDateTime.ofInstant(Instant.ofEpochMilli(timestamp), ZoneOffset.UTC);
  }